src/mvContainers.cpp

```diff
@@ -126,6 +135,10 @@ DearPyGui::fill_configuration_dict(const mvTreeNodeConfig& inConfig, PyObject* o
     checkbitset("open_on_arrow", ImGuiTreeNodeFlags_OpenOnArrow, inConfig.flags);
     checkbitset("leaf", ImGuiTreeNodeFlags_Leaf, inConfig.flags);
     checkbitset("bullet", ImGuiTreeNodeFlags_Bullet, inConfig.flags);
+    checkbitset("span_text_width", ImGuiTreeNodeFlags_SpanTextWidth, inConfig.flags);
+    checkbitset("span_available_width", ImGuiTreeNodeFlags_SpanAvailWidth, inConfig.flags);
```

Comment:
I commented them out because I couldn't properly test them and even see their effects, but I wanted to leave everything ready in case someone managed to do it... you tell me what to do

Reply:
Well, then let's comment out them in `mvContainers.cpp`, too :).